ru/lessons/basics/enum.md

```diff
@@ -1,25 +1,46 @@
 ---
-version: 0.9.0
+version: 1.2.0
 layout: page
 title: Enum
 category: basics
 order: 3
 lang: ru
 ---
 
-Набор алгоритмов для операций с коллекциями.
+Набор алгоритмов для операций с перечислениями.
```

Comment:
Isn't [`перечисляемое`](https://ru.wikipedia.org/wiki/%D0%9F%D0%B5%D1%80%D0%B5%D1%87%D0%B8%D1%81%D0%BB%D1%8F%D0%B5%D0%BC%D1%8B%D0%B9_%D1%82%D0%B8%D0%BF) closer to `enumerable` than `перечисление`? It would be nice to reconsider its usage through the whole lesson.